lib/improver/cli/nbhood_iterate_with_mask.py

```diff
@@ -35,114 +35,6 @@
 from improver import cli
 
 
-def main(argv=None):
-    """Load in arguments for applying neighbourhood processing when using a
-    mask."""
-    from improver.argparser import ArgParser
-
-    parser = ArgParser(
-        description='Apply the requested neighbourhood method via the '
-                    'ApplyNeighbourhoodProcessingWithAMask plugin to a file '
-                    'with one diagnostic dataset in combination with a file '
-                    'containing one or more masks. The mask dataset may have '
-                    'an extra dimension compared to the input diagnostic. '
-                    'In this case, the user specifies the name of '
-                    'the extra coordinate and this coordinate is iterated '
-                    'over so each mask is applied to separate slices over the'
-                    ' input data. These intermediate masked datasets are then'
-                    ' concatenated, resulting in a dataset that has been '
-                    ' processed using multiple masks and has gained an extra '
-                    'dimension from the masking.  There is also an option to '
-                    're-mask the output dataset, so that after '
-                    'neighbourhood processing, non-zero values are only '
-                    'present for unmasked grid points. '
-                    'There is an alternative option of collapsing the '
-                    'dimension that we gain using this processing using a '
-                    'weighted average.')
-    parser.add_argument('coord_for_masking', metavar='COORD_FOR_MASKING',
-                        help='Coordinate to iterate over when applying a mask '
-                             'to the neighbourhood processing. ')
-    parser.add_argument('input_filepath', metavar='INPUT_FILE',
-                        help='A path to an input NetCDF file to be processed.')
-    parser.add_argument('input_mask_filepath', metavar='INPUT_MASK_FILE',
-                        help='A path to an input mask NetCDF file to be '
-                             'used to mask the input file.')
-    parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
-                        help='The output path for the processed NetCDF.')
-    group = parser.add_mutually_exclusive_group()
-    group.add_argument('--radius', metavar='RADIUS', type=float,
-                       help='The radius (in m) for neighbourhood processing.')
-    group.add_argument('--radii-by-lead-time',
```

Comment:
This information is no longer in the docstring. I think it could be a lot clearer in the process docstrings how the radius and radii by lead time arguments work and what the expected format of the radii by lead time argument is.

Reply:
More worryingly we don't seem to have an acceptance test anywhere (for any of the neighbourhood clis) that actually tests this interface. I think we should add a test to cover this for at least one of the CLIs that have the radii by lead time functionality to show that it still works with clize. Or is this covered by @neilCrosswaite work on nbhood land sea? Does this PR depend on that PR?